catalog/app/containers/Bucket/RevisionInfo.jsx

```diff
@@ -0,0 +1,192 @@
+import cx from 'classnames'
```

Comment:
were there any significant change to this code or was it just migrated over to TS and moved to a separate file?

Reply:
There no changes to this code at all (it still js btw, not typescript). I moved it to a separate file because I imported it to another component at the first iteration. Then I changed `BucketLink` component but decided to leave `RevisoinInfo` as separate component. Because it looks like a separate component and its easier to collaborate on small components rather then big